portal/nginx.conf

```diff
@@ -37,6 +37,11 @@ http {
         listen 8000;
 
         location / {
+            root /usr/share/nginx/html;
+            try_files $uri $uri/ /usr/share/nginx/html/index.html @parcel;
```

Comment:
It seems would be easy to use stale production build locally...

Reply:
You have to run `npm run clean` :( Any thoughts on improving this? I just wanted to improve the setup so that testing production build is not too difficult.

Reply:
Maybe we can listen on another port for production build? Then, we just need to add more redirect URI and go to that port when testing production build.

Reply:
Updated the nginx configuration to use port 8010 for production build.